src/os/ui/query/queryhandler.js

```diff
@@ -8,6 +8,16 @@ goog.require('os.filter.IFilterFormatter');
 goog.require('os.filter.ISpatialFormatter');
 
 
+/**
+ * @typedef {{
+ *   entries: !Array<!Object<string, string|boolean>>,
+ *   includes: !Array<!Object<string, string|boolean>>,
+ *   excludes: !Array<!Object<string, string|boolean>>
+ * }}
+ */
+os.ui.query.activeEntries;
```

Comment:
Nit: this would be the only typedef anywhere in the application that is `camelCase` rather than `TitleCase`